File: src/main/java/br/com/newObjects/model/classes/NovaAvaliacao.java
package br.com.newObjects.model.classes;

import br.com.model.entities.classes.Avaliacao;

public class NovaAvaliacao {

    public static Avaliacao novaAvaliacao() {
        return new Avaliacao(5, NovoItemPedido.novoItemPedido(), "teste de comentário");
    }
}

File: src/test/java/br/com/newObjects/model/NovaAvaliacao.kt
package br.com.newObjects.model

import br.com.model.Avaliacao
import br.com.model.Pedido

object NovaAvaliacao {
    fun novaAvaliacao(pedido: Pedido) {
        Avaliacao(5, pedido, "teste de comentário")
    }
}
